src/components/article.js

```diff
@@ -4,7 +4,15 @@ import FlatButton from 'material-ui/FlatButton';
 
 
 const Article = (props) => {
-  const image = props.data.multimedia[0] ? props.data.multimedia[4].url : null;
+  let image = null;
+  if (props.data.multimedia.length > 0) {
```

Comment:
Thanks @Ellisande 
I'm actually going with a slight variation because I actually want to cap it at a max resolution using the 4th highest image, and only select the highest available in the array if 4 isn't available. (The array sorts image resolution from lowest to highest, and it gets insanely high. So trying to balance.)
```jsx
  const { multimedia } = props.data;
  let image;
  if (multimedia.length > 0) {
    image = multimedia[4].url || multimedia.slice(-1).url;
  }
```